.github/workflows/general.yml

```
@@ -55,6 +55,20 @@ jobs:
           run: cargo test --release -- --include-ignored
         - name: Run fuzzer
           run: cargo run --release --bin=fuzzer
+  check_nix:
+    name: Check nix code
+    strategy: 
+      matrix: 
```

nitpick: can we drop the trailing spaces?
```suggestion
    strategy:
      matrix:
```